src/blueprints/organization/static/js/charts.js

```diff
@@ -1,34 +1,24 @@
-// Bar chart with custom tooltip
-const dataBarCustomTooltip = {
-    type: 'bar',
+window.onload = formChart
+function formChart() {const xValues = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  const yValues = [8, 25, 15, 10, 22, 33, 48];
+  const barColors = ["red", "green","blue","orange","brown", "purple", "black"];
```

Comment:
Suggestion: Bars should be uniform in color

Reply:
This is a proof of functionality, design aspects will follow. I will be sure to change it.